### DISCUSSION TITLE
Is it possible to Extend the product view

### DISCUSSION THREAD

**Discussion by ahgamal-ms** [microsoft/Dynamics365Commerce.Solutions#285] [Q&A] [answered]
Extend columns in the Product views. Currently, we can see that the custom columns appear correctly in the SearchView, but do not appear in the product category listing. Is it possible to extend the below view

I have tried to search on it on [Extend POS views to add custom columns and app bar buttons - Commerce | Dynamics 365 | Microsoft Learn](https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/pos-view-extension) but could not find any reference for below view

![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/74003164/a51104a6-7b7e-469a-9845-47525b85a508)

**Comment by madyke** [comment, marked as answer]
Hi @ahgamal-ms , unfortunately extending the columns on the ProductView is not currently supported. If this is a business requirement for a customer, you'll need to submit an extension request so that an engineer can review and triage it:
[Extensibility requests](https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/extensibility/extensibility-requests)
